store/mockstore/unistore/cophandler/cop_handler.go

```diff
@@ -301,34 +401,15 @@ func buildResp(chunks []tipb.Chunk, closureExecutor *closureExecutor, ndvs []int
 	}
 	executors := dagReq.Executors
 	if dagReq.CollectExecutionSummaries != nil && *dagReq.CollectExecutionSummaries {
+		// for simplicity, we assume all executors to be spending the same amount of time as the request
```

Comment:
Looks like before this pr, the `executor_summary` has a meaningfull value for each executor, is it ok to change it to a fake value?

Reply:
yes, the executor_summary is mostly for `explain analyze`, it is okay on unistore because it is mostly used for testing purposes. more over the result is not wrong and is not fake, it is just the same for all executors, i.e. coarse-grained than the original version.